src/get-all-static-tiles.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const getTileUrl = require('../utils/get-tile-url');
+const getStaticTile = require('./get-static-tile');
+
+function getAllTiles(tileImgProperties, { style, accessToken }) {
+  const tiles = tileImgProperties.tiles;
+  const promises = [];
+  tiles.forEach((tile) => {
```

Comment:
```suggestion
  tiles.forEach(tile => {
```

Reply:
Hmm, I get a linting error from our Mapbox config when I do this 🤔

https://github.com/mapbox/eslint-config-mapbox/blob/master/base.js#L15

Reply:
Oh! I guess we require parens. Let's just close these suggestions 🙈